lib/spack/spack/cmd/external.py

```diff
@@ -150,7 +156,7 @@ def external_find(args):
 
 
 def packages_to_search_for(
-    *, names: Optional[List[str]], tags: List[str], exclude: Optional[List[str]]
+    *, names: Optional[List[str]], tags: List[str], exclude: Optional[List[str]], dependencies_only: Optional[bool]
```

Comment:
Does `dependencies_only` mean that `spack external find --dependencies x` will not look for `x` but just deps of it? IMO `--dependencies` should pull in both (and this parameter should be renamed).